lexik/cron-file-generator-bundle/1.0/config/packages/lexik_cron_file_generator.yaml

```diff
@@ -0,0 +1,13 @@
+# see https://github.com/lexik/LexikCronFileGeneratorBundle/blob/master/Resources/doc/index.md
+lexik_cron_file_generator:
+    env_available:                 # declare your env availables
+        - staging                    # example: staging, prod
+    user:
+        staging: 'project_staging'     # user for execute the cron
+    php_version: 'php7.2'
+    absolute_path:
+        staging: 'project_dir/bin/console'
```

Comment:
Do you mean `'%kernel.project_dir%/bin/console'`?

Reply:
No it's juste a path on the server, not with symfony